### ISSUE TITLE
i18n-widget: No label for headings

### ISSUE THREAD

**Issue by mal-tee** [datenanfragen/website#855]
I tried to use our homepage with NVDA on Win 10 with FF.

![grafik](https://user-images.githubusercontent.com/45886910/154838437-ac329370-0119-4a03-a36d-d307b84f33c8.png)

It skips directly to "Deutsch"/"alle" without reading "Sprache" or "Land", making it quite confusing.

I think this occurs because we don't provide a `<label>`?

**Comment by baltpeter** [comment]
Not sure why it doesn't read the headings but yes, those should absolutely be labels for the `<select>`s.

**Comment by mal-tee** [comment]
Addendum: It does read the headings when browsing with the arrow keys, but doesn't if you jump to the select via Tab.
